src/browser/modules/Stream/CypherFrame/VisualizationView.tsx

```diff
@@ -119,8 +119,11 @@ export class Visualization extends Component<any, VisualizationState> {
     return new Promise((resolve, reject) => {
       this.props.bus &&
         this.props.bus.self(
-          CYPHER_REQUEST,
-          { query: query, queryType: NEO4J_BROWSER_USER_ACTION_QUERY },
+          ROUTED_CYPHER_READ_REQUEST,
```

Comment:
Risk on consistency, as the read might go to a core that's not got the write update from the main query yet